sequencer/src/state.rs

```diff
@@ -90,6 +94,14 @@ impl ValidatedState {
         self.fee_merkle_tree.update(account, amount).unwrap();
     }
 
+    pub fn balance(&mut self, account: FeeAccount) -> Option<FeeAmount> {
```

Comment:
This function is only used in tests (unless my rust-analyzer is tripping). I'm not sure returning `None` is a good idea for the `NotInMemory` case, but if we make it test code only then I think it's not a problem.

Reply:
I know we don't have a stable public Rust API yet but I feel like a function like this should be part of it. Could easily be useful outside tests for clients and apps built on top of this. What do you think such a public function should do in the `NotInMemory` case? Should it return `Result` instead of `Option`?